### PR TITLE
Display User's Bid List 

### PR DESCRIPTION
**Pull request by mjoyce91**
Addresses #642 and #643 by adding a section to the Profile that displays all positions the user has added to their bid list. Each results card includes a link to the position and a button to delete the position from their bid list.

### CODE REVIEW

File: src/Components/BidList/BidList.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+bidListToggleIsLoading, bidListToggleSuccess }) => {
+  const showError = !bidListToggleIsLoading && !bidListToggleSuccess
+      && bidListToggleHasErrored;
+  const showSuccess = !bidListToggleIsLoading && bidListToggleSuccess
+      && !bidListToggleHasErrored;
+  const isLoading = bidListIsLoading && !bidListHasErrored;
+  return (
+    <div
+      className={`usa-grid-full saved-searches-container
+    ${bidListIsLoading ? 'results-loading' : ''}`}
```
Comment (burgwyn) [inline src/Components/BidList/BidList.jsx]: Pull the ternary out to a const

File: src/Components/BidList/BidList.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    >
+      <ProfileSectionTitle title="Your Bid List:" />
+      {
+        // Deleting a bid has errored
+        showError &&
+          <Alert
```
Comment (burgwyn) [inline src/Components/BidList/BidList.jsx]: Let's pull these Alerts into a separate function and then `{ showAlert() }`

File: src/Components/ProfilePage/__snapshots__/ProfilePage.test.jsx.snap [hunk 3]
```diff
@@ -5,7 +5,7 @@ exports[`ProfilePageComponent matches snapshot 1`] = `
   className="usa-grid-full"
 >
   <h1>
-    Hello, john!
+    Hello, ...!
```
Comment (burgwyn) [inline src/Components/ProfilePage/__snapshots__/ProfilePage.test.jsx.snap]: Is this correct?

Comment (mjoyce91) [reply src/Components/ProfilePage/__snapshots__/ProfilePage.test.jsx.snap]: Used the wrong object when I abstracted the props. See updates.

**Comment by mjoyce91** [comment]
See updates

**Pull request merged**